galley/pkg/fs/fssource_test.go

```diff
@@ -105,13 +105,31 @@ spec:
     - destination:
         host: some.example.internal
 `
-var fst *fsTestSourceState
+var (
+	fst *fsTestSourceState
+)
 
 type fsTestSourceState struct {
 	ConfigFiles map[string][]byte
 	rootPath    string
 }
 
+type scenario struct {
```

Comment:
Can you please add comments to each field describing the intent? Otherwise looks ok to me.

Reply:
comments add~